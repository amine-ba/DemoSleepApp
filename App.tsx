import React from "react";
import { Center, NativeBaseProvider } from "native-base";
import DailyScoreFormScreen from "./src/screens/DailyScoreFormScreen";

export default function App() {
  return (
    <NativeBaseProvider>
      <Center px={4} flex={1}>
        <DailyScoreFormScreen />
      </Center>
    </NativeBaseProvider>
  );
}
