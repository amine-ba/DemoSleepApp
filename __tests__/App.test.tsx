import React from "react";
import { render } from "@testing-library/react-native";

import App from "../App";

describe(" <App /> Component ", () => {
  it(" should successfully rendred ", () => {
    const appTree = render(<App />).toJSON();
    expect(appTree).toMatchSnapshot();
  });
});
