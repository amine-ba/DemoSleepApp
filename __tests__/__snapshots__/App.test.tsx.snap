// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` <App /> Component   should successfully rendred  1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
